README.md

```diff
@@ -164,12 +162,153 @@ outcome of an authorisation attempt using different configurations of
 user/record pairs based on application state. These variations should be
 organised into separate RSpec context blocks, as in the previous example.
 
+## Testing For an Allow List of Permitted Actions
+
+As of Pundit Matchers 2.1, the recommended approach is to use an explicit
+allow list of actions permitted by the policy (`permit_only_actions`) which
+will cause the test to fail if other actions in the policy are not forbidden.
+
+Using allow lists ensures for comprehensiveness so that any actions that
+shouldn't be permitted don't slip through the cracks in practice, for example
+due to developer oversight. The [OWASP Top 10 - 2021][owasp-top-10] recommends
+that access control enforces policy so that users cannot act outside of their
+intended permissions. It was discovered that 94% of applications that were
+tested had some form of [broken access control][broken-access-control] with an
+average incidence rate of 3.81%.
+
+The following example tests a policy that authorises visitors to view articles
+only, while administrators can create and update articles (but are forbidden
+from deleting them):
+
+```ruby
+require 'rails_helper'
+
+describe ArticlePolicy do
+  subject { described_class.new(user, article) }
+
+  let(:article) { Article.new }
+
+  context 'being a visitor' do
+    let(:user) { nil }
+
+    it { is_expected.to permit_only_actions(%i[index show]) }
+  end
+
+  context 'being an administrator' do
+    let(:user) { User.new(administrator: true) }
+
+    it { is_expected.to permit_only_actions(%i[index show create update]) }
+  end
+end
+```
+
+This approach would add be robust in the following scenario:
+
+* ArticlePolicy is created with index, create, destroy
+* The policy is tested with `permit_only_actions(%i[index create destroy])`
+* After 8 months, a developer adds publish action, which would automatically
+  be forbidden since it is not in the allow list of permitted actions.
```

Comment:
```suggestion
* `ArticlePolicy` is created with `index`, `create`, and `destroy` actions
* The policy is tested with `permit_only_actions(%i[index create destroy])`
* After several months, a developer adds the `publish` action, which would automatically
  be tested to be forbidden since it is not in the allow list of permitted actions.
```

Reply:
I've added these improvements.